source/MapPanel.cpp

```diff
@@ -67,6 +69,7 @@ namespace {
 	const std::string SHOW_STORED_OUTFITS = "Show stored outfits on map";
 	const unsigned MAX_MISSION_POINTERS_DRAWN = 12;
 	const double MISSION_POINTERS_ANGLE_DELTA = 30.;
+	static const int MAX_STARS = 5;
```

Comment:
Everything in an anonymous namespace is implicitly static.
```suggestion
	const int MAX_STARS = 5;
```